packages/lab/src/components/Flow/Node/Node.tsx

```diff
@@ -0,0 +1,192 @@
+  /** Node color */
+  color?: HvColorAny;
+  /** Node expanded */
+  expanded?: boolean;
+  /** Node inputs */
+  inputs?: HvFlowNodeInput[];
+  /** Node outputs */
+  outputs?: HvFlowNodeOutput[];
+  /** Node parameters */
+  params?: HvFlowNodeParam[];
+  /** A Jss Object used to override or extend the styles applied to the component. */
+  classes?: HvFlowNodeClasses;
+}
+
+const isInputConnected = (id, type, idx, edges) => {
+  if (type === "target") {
+    return edges.some(
+      (e) => e.target === id && e.targetHandle === idx.toString()
+    );
+  }
+  if (type === "source") {
+    return edges.some(
+      (e) => e.source === id && e.sourceHandle === idx.toString()
+    );
+  }
+};
+
+export const HvFlowNode = ({
+  id,
+  type,
+  title,
+  description,
+  color: colorProp = theme.colors.cat3_40,
+  expanded = false,
+  params,
+  classes: classesProp,
+  className,
+}: HvFlowNodeProps) => {
+  const [showParams, setShowParams] = useState(expanded);
+  const reactFlowInstance = useReactFlow();
+
+  const { classes, cx, css } = useClasses(classesProp);
+  const color = getColor(colorProp);
+
+  const { nodeGroups, nodeTypes } = useFlowContext();
+  const edges = useStore((s) => s.edges);
+  const nodes = useStore((s) => s.getNodes());
+
+  const node = nodes.find((n) => n.id === id);
+
+  const groupId = nodeTypes?.[type].meta?.groupId;
+  const inputs = nodeTypes?.[type]?.meta?.inputs;
+  const outputs = nodeTypes?.[type]?.meta?.outputs;
+  const icon = groupId && nodeGroups && nodeGroups[groupId].icon;
+
+  useEffect(() => {
+    const newNodes = nodes.map((n) => {
+      if (n.id === id) {
+        if (Object.keys(n.data).length === 0) {
+          params?.forEach((param) => {
+            n.data[param.label] = param.value;
+          });
+        }
+      }
+      return n;
+    });
+    reactFlowInstance.setNodes(newNodes);
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, []);
+
+  const hasParams = !!(params && params.length > 0);
+
+  return (
+    <div
+      className={cx(
+        css({ border: `1px solid ${color}` }),
+        classes.root,
+        className
+      )}
+    >
+      <div
+        className={cx(css({ backgroundColor: color }), classes.headerContainer)}
+      >
+        <div className={classes.titleContainer}>
+          {icon}
+          <HvTypography variant="title4" className={classes.title}>
+            {groupId}
+          </HvTypography>
+        </div>
+        <div style={{ display: "flex" }}>
+          <HvTooltip title={<HvTypography>{description}</HvTypography>}>
+            <div>
+              <Info />
+            </div>
+          </HvTooltip>
+          <HvButton
+            icon
+            onClick={() => setShowParams((p) => !p)}
+            style={{
+              cursor: hasParams ? "pointer" : "not-allowed",
+            }}
+          >
+            <Down />
+          </HvButton>
+        </div>
+      </div>
+      <div className={classes.descriptionContainer}>
```

Comment:
I think we should rename this `descriptionContainer` class since this is wrapping the `title`.  It can be confusing since we have a `description` prop that we use above.